Nona-Creative/nona-playground
# feat(react): add sentry and sentry middleware example

![](https://media.giphy.com/media/YWht0oD4mxNDO/giphy.gif)

## Scope

PT5 - Sentry - Redux Sentry Middleware

## Work Done

- Add Sentry and Redux Sentry Middleware to web-ui-react

## Steps to Test

- pull the branch
- create an environment file and add a Sentry DSN environment file.
- Cause an application error and check that it captures the redux application state in Sentry
